services/brig/src/Brig/API/Internal.hs

```diff
@@ -334,20 +336,39 @@ changeSelfEmailMaybeSend u DoNotSendEmail email = do
     ChangeEmailIdempotent -> pure ChangeEmailResponseIdempotent
     ChangeEmailNeedsActivation _ -> pure ChangeEmailResponseNeedsActivation
 
-listActivatedAccountsH :: JSON ::: Either (List UserId) (List Handle) -> Handler Response
-listActivatedAccountsH (_ ::: qry) = do
-  json <$> lift (listActivatedAccounts qry)
-
-listActivatedAccounts :: Either (List UserId) (List Handle) -> AppIO [UserAccount]
-listActivatedAccounts = \case
-  Left us -> byIds (fromList us)
-  Right hs -> do
-    us <- mapM (API.lookupHandle) (fromList hs)
-    byIds (catMaybes us)
+listActivatedAccountsH :: JSON ::: Either (List UserId) (List Handle) ::: Bool -> Handler Response
+listActivatedAccountsH (_ ::: qry ::: includePendingInvitations) = do
+  json <$> lift (listActivatedAccounts qry includePendingInvitations)
+
+listActivatedAccounts :: Either (List UserId) (List Handle) -> Bool -> AppIO [UserAccount]
+listActivatedAccounts elh includePendingInvitations =
+  case elh of
+    Left us -> byIds (fromList us)
+    Right hs -> do
+      us <- mapM (API.lookupHandle) (fromList hs)
+      byIds (catMaybes us)
   where
-    byIds uids =
-      filter (isJust . userIdentity . accountUser)
-        <$> API.lookupAccounts uids
+    byIds :: [UserId] -> AppIO [UserAccount]
+    byIds uids = catMaybes <$> (API.lookupAccounts uids >>= traverse accountValid)
+
+    accountValid :: UserAccount -> AppIO (Maybe UserAccount)
+    accountValid account = case (userIdentity . accountUser $ account) of
+      Nothing -> pure Nothing
+      Just ident
+        | accountStatus account == PendingInvitation ->
+          if includePendingInvitations
+            then case emailIdentity ident of
+              Nothing -> pure (Just account) -- should not happend, scim-invited user always have an email
```

Comment:
```suggestion
              Nothing -> pure (Just account)
```

This can happen, because this function is called in situations other then scim-invite.

Reply:
This case has additional `accountStatus account == PendingInvitation` condition

Reply:
oops, you're right.  never mind.